config/webpacker.yml

```diff
@@ -25,10 +25,10 @@ development:
   <<: *default
 
 # uncomment these to enable webpack:server
-#  dev_server:
-#    host: 0.0.0.0
-#    port: 8080
-#    https: false
+  dev_server:
```

Comment:
EDIT: never mind, webpacker 2.0 changed this, it will work until you actually run the dev-server. It will only break after that :).